Source/EasyNetQ/PullingConsumerFactory.cs

```diff
@@ -29,22 +30,22 @@ public interface IPullingConsumerFactory
     /// <inheritdoc />
     public class PullingConsumerFactory : IPullingConsumerFactory
     {
+        private readonly IConsumerConnection connection;
         private readonly IPersistentChannelFactory channelFactory;
         private readonly IMessageSerializationStrategy messageSerializationStrategy;
         private readonly IProduceConsumeInterceptor produceConsumeInterceptor;
 
         /// <summary>
         ///     Creates PullingConsumerFactory
         /// </summary>
-        /// <param name="channelFactory">The channel factory</param>
-        /// <param name="produceConsumeInterceptor">The produce-consume interceptor</param>
-        /// <param name="messageSerializationStrategy">The message serialization strategy</param>
         public PullingConsumerFactory(
+            IConsumerConnection connection,
```

Comment:
The same is here.

https://github.com/EasyNetQ/EasyNetQ/pull/1273/files#r710740185